### PR TITLE
Bug 1815631 - changed height and reduced padding for `Active` text from the Studies menu

### PR DESCRIPTION
**Pull request by gitstart**
The PR fixes the `Active` text truncate issue on Studies screen.

### Steps to reproduce

1. Open the app and restart it once.
2. Enable the Debug menu by tapping the Firefox icon from Settings > About Firefox 5 times.
3. Enroll in an experiment by tapping a branch from Settings > Nimbus Experiments > experiment name.
4. Go to Settings > Data collection > Studies.
5. Observe the “Active” section.

### Issue video
https://www.loom.com/share/ddda083d88a0449198eac73868969a76

### Demo video after fix
https://www.loom.com/share/6875b27b4349474aae8d34ce22ee998c

### Pull Request checklist
- [ ] **Tests:** This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots:** This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility:** The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

**QA**
- [x] QA Needed
### To download an APK when reviewing a PR (after all CI tasks finished running):

1. Click on Checks at the top of the PR page.
2. Click on the firefoxci-taskcluster group on the left to expand all tasks.
3. Click on the build-debug task.
4. Click on View task in Taskcluster in the new DETAILS section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1815631

### CODE REVIEW

File: fenix/app/src/main/res/layout/studies_section_item.xml [hunk 1]
```diff
@@ -22,12 +22,12 @@
         android:layout_marginStart="@dimen/top_bar_alignment_margin_start"
         android:id="@+id/title"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/section_header_height"
+        android:layout_height="wrap_content"
```
Comment (Mugurell) [inline fenix/app/src/main/res/layout/studies_section_item.xml:25]: Nice catch!
We see now that using a hardcoded size for layout of a text should be avoided.

**Review by Mugurell — CHANGES_REQUESTED** [review]
Thank you for helping with this!
Nice catch with the `wrap_content` attribute but the padding update should use different values.

For future references, this screen was implemented in https://github.com/mozilla-mobile/fenix/issues/20764.
If we inspect Figma linked there we see that:
- the section title should have
  - 12sp size
  - 600 weight - semibold (the current `bold` weight has a value of 700w)
- the section title's top _margin_ should be 8dp
- the sections title's bottom _margin_ should be 14 dp

@gitstart If you are planning to update the layout can you make the above changes?
cc @Amejia481

**Comment by gitstart** [comment]
Hello @Mugurell,🙏
Requested changes pushed.🚀

**Comment by Mugurell** [comment]
@Mergifyio backport releases_v111

**Pull request merged**